src/azure-cli/azure/cli/command_modules/vm/manual/commands.py

```diff
@@ -21,5 +25,34 @@ def load_command_table(self, _):
         operations_tmpl='azure.mgmt.compute.operations._ssh_public_keys_operations#SshPublicKeysOperations.{}',
         client_factory=cf_ssh_public_key,
     )
+
+    vm_gallery_sharing_profile = CliCommandType(
+        operations_tmpl=(
+            'azure.mgmt.compute.operations._gallery_sharing_profile_operations#GallerySharingProfileOperations.{}'
+        ),
+        client_factory=cf_gallery_sharing_profile,
+        operation_group='galleries'
+    )
+
     with self.command_group('sshkey', vm_ssh_public_key, client_factory=cf_ssh_public_key) as g:
         g.custom_command('create', 'sshkey_create')
+
+    with self.command_group('sig shared-gallery', vm_shared_gallery, client_factory=cf_shared_gallery,
+                            is_experimental=True, operation_group='shared_galleries') as g:
+        g.custom_command('list', 'sig_shared_gallery_list')
+
+    with self.command_group('sig share', vm_gallery_sharing_profile, operation_group='shared_galleries',
+                            is_experimental=True, client_factory=cf_gallery_sharing_profile, min_api='2020-09-30') as g:
+        g.custom_command('add', 'sig_share_update', supports_no_wait=True)
+        g.custom_command('remove', 'sig_share_update', supports_no_wait=True)
+        g.custom_command('reset', 'sig_share_reset', supports_no_wait=True)
```

Comment:
because we support no wait here, we need to add wait command to provide a way for customer query existing operation status.

Reply:
If I add a wait command, then linter will throw an err
```
The operation 'azure.mgmt.compute.v2020_09_30.operations._gallery_sharing_profile_operations#GallerySharingProfileOperations.get' is invalid.
Traceback (most recent call last):
  File "c:\users\xiaofanzhou\zxf\azure-cli\src\azure-cli-core\azure\cli\core\commands\command_operation.py", line 61, in get_op_handler
    handler = getattr(handler, part)
AttributeError: type object 'GallerySharingProfileOperations' has no attribute 'get'
```

Reply:
Added